packages/webpack-plugin/src/compiler.ts

```diff
@@ -127,15 +122,15 @@ function compileVanillaSource(
           library: {
             type: 'commonjs2',
           },
-          import: [`!!${request}`],
+          import: [loader.resourcePath],
```

Comment:
This change is unrelated to the fix but is an improvement that we couldn't make before due to the old approach.